gtk4/sample/examples/application5/README.md

```diff
@@ -0,0 +1,39 @@
+# Step 5: Add a GSettings object
+
+The original files are located in the following directory.
+
+- https://gitlab.gnome.org/GNOME/gtk/-/blob/main/examples/application5
+
+The original schema file is [here](https://gitlab.gnome.org/GNOME/gtk/-/blob/main/examples/application5/org.gtk.exampleapp.gschema.xml) and the license is LGPL 2.1 or later.
+
+This program uses GSettings object of the GIO library.
+You need to compile the schema file (`org.gtk.exampleapp.gschema.xml`) in advance.
+Therefore, there are two tasks.
+
+- Compile the schema file
+- Run the Ruby script `exampleapp.rb`
+
+These tasks are integrated into a Rakefile.
+What you need is just to type `rake`.
+
+```console
+$ cd gtk4/sample/examples/application5
+$ rake
+```
+
+If you want to give filenames to `exampleapp.rb`, use square bracket.
+
+```console
+$ rake default[README.md,Rakefile]
```

Comment:
Ah, we need to specify one ore more arguments to `exampleapp.rb`...

The Rake syntax isn't easy to use.
How about the followings?

1. We remove the `run` task and users run `exampleapp.rb` directly: `ruby exampeapp.rb README.md Rakefile`
2. `exampleapp.rb` sets `GSETTINGS_SCHEMA_DIR` internally.
3. `exampleapp.rb` checks `File.exist?(File.join(__dir__, "gschema.compiled"))` and reports an error when the check fails: `gschema.compiled doesn't exist. Run "rake" to generate tit.` or something